### ISSUE TITLE
Similarity detection seems to be working inaccurately 

### ISSUE THREAD

**Issue by bernardsilenou** [hzi-braunschweig/SORMAS-Project#3166]
### Bug Description
Users reported that the  system suggested unrelated persons during case and contact import no matter the value set for name similarity threshold. System recommended persons with different names. This occurred on a production server on 1,48,2 so did not try to reproduce the bug.
### Steps to Reproduce
1. Import cases and contacts  using a server on  version 1,48,2 
2.

### System Details
* Device: web
* SORMAS version: 1,48,2
* Android version/Browser:
* Server URL:
* User Role:

**Comment by bernardsilenou** [comment]
A similar issue like this one occurred on the past and was resolved: \issue #2764

**Comment by bernardsilenou** [comment]
@MartinWahnschaffeSymeda @markusmann-vg this issue was mentioned in the performance meeting today

**Comment by MateStrysewske** [comment]
Don't have any problems with it, neither during import nor during case creation. We need more details about the threshold that has been configured on the server.

Do they have the same problem during case creation or does it only happen during import?

**Comment by bernardsilenou** [comment]
@MateStrysewske I have not yet tested myself but they reported that this occurred during import. They used 0,95 but q1 similarity as applied by sormas used. I would send a sample screenshot by email to you.

**Comment by MateStrysewske** [comment]
"They used 0,95 but q1 similarity as applied by sormas used." - I don't really get what you're trying to say here. Could you rephrase that sentence, please? Ideally please send me a screenshot of the properties file, specifically the property that defines the similarity threshold.

**Comment by bernardsilenou** [comment]
@MateStrysewske I have sent you the sormas.properties file by email
